plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/ui/views/properties/PropertiesView.java

```diff
@@ -106,7 +106,9 @@ public void selectTab(String label) {
 		}catch(Exception ex) {
 			//probably not rendered yet
 		}
-		new WaitUntil(new AnotherTabsRendered(old), TimePeriod.NORMAL, false);
+		if (!old.contains(label)) {
```

Comment:
'old' is List and my goal is to check whether the list already contains my label